Sources/GDSCommon/Extensions/UIKit/UIViewController+Extensions.swift

```diff
@@ -2,7 +2,10 @@ import UIKit
 
 extension UIViewController {
     func setBackButtonTitle(isHidden: Bool = false) {
-        navigationItem.backButtonTitle = NSLocalizedString(key: "backButtonTitle")
+        let backButtonTitle = GDSLocalisedString(stringKey: "GDSCommonBackButton",
+                                                 bundle: .module)
+        let backButton = UIBarButtonItem(title: backButtonTitle.value, style: .plain, target: nil, action: nil)
+        navigationController?.navigationBar.topItem?.backBarButtonItem = backButton
```

Comment:
@robinphillips I tested this implementation with `navigationItem.backButtonTitle` and it was never actually updating the back title in the demo app. When ive changed to set the `backBarButtonItem` it works as expected. Would be good if you could just sanity check this locally as I know we had a chat about this

Reply:
I'll take a look. It is working in the gov uk app using GDS Common screens. I've just checked. But I'll have a look at the demo app now.

Reply:
**Just to summarise Robin and my huddle:**

`navigationItem.backButtonTitle` does work when there is no navigationTitle on the previous page

Question for the team, would it be reasonable to say if we conform to BaseViewController, we are asking for the back button to be "Back" In English/welsh? And then if we wanted the back button to have the navigationTitle of the previous page, we would just not conform to BaseViewController?

Reply:
I think _navigationTitle_ of the previous page should _**always**_ be the default behaviour unless the string is too long, when it change to back. 

This is the expected design pattern on iOS and how users will expect our app to work.

Reply:
@benmckillop, I think everything is working as it should and some of the changes here can be reverted aside from adding the `localizable.strings` file and adjusting string initialisation to:
`navigationItem.backButtonTitle = NSLocalizedString(key: "backButtonTitle", bundle: .module)`

However, I think to do what I think you want (have a custom string for the back button), you would need to do:
`navigationItem.backButtonTitle = NSLocalizedString(key: "someStringKey")` 
on your previous screen.

This should then appear on the subsequent screen as the back button title.

Reply:
Ignoring Oli's comment for now (As I will bring this to design on Monday) Is there anything wrong with the new way I've implemented it? It does work, and makes much more sense than setting it on the screen before. I think that would be less readable and more complicated, especially if it was a change we wanted in GDSCommon rather than just in Wallet.

Reply:
Or do you potentially mean that I just set that locally in Wallet? If so would the fact the screen I want it on conforms to BaseViewController is an issue I think

Reply:
Hmmm okay, happy to chat this through. I think another thing we now have to consider is that Oli mentions we should be not setting this button to back, but conformance to BaseViewController will set this to back by default